questions/106-medium-trimleft/template.ts

```diff
@@ -1 +1,3 @@
-export type TrimLeft<S extends string> = any
+export type TrimLeft<S extends string> = S extends `${ ' ' | '\n' | '\t' }${ infer Rest }` 
```

Comment:
You can define an extra type for this right?

type Whitespace = ' ' | '\n' | '\t';

export type TrimLeft<S extends string> = S extends `${Whitespace}${infer Rest}`
  ? TrimLeft<Rest>
  : S;

Reply:
Yeah, I can define an extra type for better readability and be able to expand the set of whitespace characters if I need it.